evergreen/evergreen.yml

```diff
@@ -252,6 +267,7 @@ functions:
         env:
           PACKAGES_SOURCE: ${PACKAGES_SOURCE}
           PACKAGES_SOURCE_KEY: ${PACKAGES_SOURCE_KEY}
+          NUGET_SIGN_CERTIFICATE_FINGERPRINT: ${$NUGET_SIGN_CERTIFICATE_FINGERPRINT}
```

Comment:
`${$NUGET_SIGN_CERTIFICATE_FINGERPRINT}` correct ?

Reply:
Good catch! Thank you!